scripts/language_model/run_squad.py

```diff
@@ -82,7 +82,7 @@
     '--uncased',
     action='store_true',
     help=
-    'if set, inputs are converted to lower case. Up to 01/04/2020, all released models are cased'
+    'if set, inputs are converted to lower case. Up to 04/01/2020, all released models are cased'
```

Comment:
Some countries default to MM/DD, others DD/MM. Let's use January 4th 2020 to be clear